Project.toml

```diff
@@ -12,7 +12,7 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
 FastClosures = "0.2.1, 0.3.0"
-LinearOperators = "2.0"
+LinearOperators = "2.0, 2.2"
```

Comment:
Is it suppose to be working with `2.1` also? If so, could we say `LinearOperators = "2"` instead of listing the releases?